HabitLogger.gkemeza/HabitLogger.gkemeza/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using HabitLogger;
+
+internal class Program
+{
+    public string operationsTable = "Operations";
+    public string habitsTable = "Habits";
+
+    private static void Main(string[] args)
+    {
+        Methods methods = new Methods();
+
+        if (!File.Exists("HabitLogger.db"))
+        {
+            methods.CreateTwoTables();
+
+            methods.GenerateHabits();
+
+            methods.InsertRandomRecords(1);
+            methods.InsertRandomRecords(2);
+        }
+
+        int habitId;
+
+        while (true)
+        {
+            Console.WriteLine("Choose a habit or create new:");
+            Console.WriteLine("0. Create a new habit");
+
+            methods.ShowAllHabits();
+
+            string input2 = Console.ReadLine();
```

Comment:
I see you use the variable `input2` and `input` quite a lot. When I see it, it's not immediately clear what kind of input. It would be nice if the input says something like: `inputIdToDelete` or `inputNewHabitName`. Stuff like that, try to be very verbose, so the code is easier to understand.